packages/autocomplete-core/src/utils/isChrome.ts

```diff
@@ -0,0 +1,14 @@
+const chromeRegex = /(?:crmo|crios|chrome)/i;
+const edgeRegex = /edg(?:e|ios|a)?/i;
```

Comment:
We don't need to make it a non-capturing group (since we don't capture anything, we just match) and this shaves a few bytes.

```suggestion
const chromeRegex = /(crmo|crios|chrome)/i;
const edgeRegex = /edg(e|ios|a)?/i;
```